python/tests/test_strategies.py

```diff
@@ -20,14 +21,16 @@ def test_backtest():
     """
     Ensures that the backtest function works on all the registered strategies, with their default parameter values
     """
-    current_date = datetime.today().strftime("%Y-%m-%d")
     sample = pd.read_csv(SAMPLE_CSV, parse_dates=["dt"])
+    # Simulate custom indicator
+    sample["custom"] = np.random.random((sample.shape[0],)) * 100
+
     for strategy in STRATEGY_MAPPING.keys():
         if strategy == "sentiment":
-            data = get_yahoo_data("TSLA", "2020-01-01", current_date)
+            data = get_yahoo_data("TSLA", "2020-01-01", "2020-07-04")
```

Comment:
I think hardcoded end date in sentimentstrategy, better stick with `current_date`. if proven that this works and the future builds will not fail, I am ok with it.

Reply:
Changed it to a static date so that the date range being tested is the same over time (similar to the other strategy tests). Also confirmed that the build passes :smile: